TeamCode/src/main/java/org/firstinspires/ftc/teamcode/subsystems/DriveSystem.java

```diff
@@ -17,13 +17,13 @@ public DriveSystem(Motor frontL, Motor frontR, Motor backL, Motor backR){
 
         drive = new MecanumDrive(fL, fR, bL, bR);
     }
-    //not in current use and yuou cant make me get rid of it
+    //not in current use and you cant make me get rid of it
```

Comment:
<img src="https://user-images.githubusercontent.com/17654209/99135070-8fcdb780-25ee-11eb-8f80-aeb515b7b5a8.png" width="50px" title="bruh" />

Reply:
the spelling bothered me